test/Libraries/AnalysisTests/LabelTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Analysis;
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Interfaces;
+using Dynamo.Visualization;
+using NUnit.Framework;
+
+namespace AnalysisTests
+{
+    [TestFixture]
+    class LabelTests
+    {
+        /// <summary>
+        /// This test method will execute the next methods from the Label class:
+        /// private Label(Point point, string label)
+        /// public static Label ByPointAndString(Point point, string label)
+        /// public void Tessellate(IRenderPackage package, TessellationParameters parameters)
+        /// </summary>
+        [Test, Category("UnitTests")]
+        public void LabelByPointAndStringTest()
+        {
+            //Arrange
+            var labelTest = Label.ByPointAndString(Point.ByCoordinates(100,100), "TestingLabel");
+            var factory = new DefaultRenderPackageFactory();
+            var package = factory.CreateRenderPackage();
+
+            //Act
+            labelTest.Tessellate(package, new TessellationParameters());
+
+            //Assert
+            Assert.IsNotNull(labelTest);
```

Comment:
I think this assertion should be before the call to `Tessellate`. Also, can you check if certain properties of `Label` are set in addition to checking for it to be non-null?

Reply:
@aparajit-pratap 
This depends of the previous comment, if I delete the call to Tessellate method there is no need to do it, otherwise I will move it before the call.